### PR TITLE
Fix: Fixed a typo and added a link to the Builders page

### PR DESCRIPTION
**Pull request by pachicodes**
## Description

Fixed a typo and added a link to the Builders page

## Reviewer Notes

If there are links or steps needed to test this work, add them here.

## Related Issue(s) / Ticket(s)

If there are any related [GitHub Issues](https://github.com/newrelic/developer-website/issues) or JIRA tickets, add links to them here.
* [issue]()

## Screenshot(s)

If relevant, add screenshots here.

## Use Conventional Commits

Please help the maintainers by leveraging the following [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/)
standards in your pull request title and commit messages.

## Use `chore`

* for minor changes / additions / corrections to content.
* for minor changes / additions / corrections to images.
* for minor non-functional changes / additions to github actions, github templates, package or config updates, etc

```bash
git commit -m "chore: adjusting config and content"
```

## Use `fix`

* for minor functional corrections to code.

```bash
git commit -m "fix: typo and prop error in the code of conduct"
```

## Use `feat`

* for major functional changes or additions to code.

```bash
git commit -m "feat(media): creating a video landing page"
```

### CODE REVIEW

File: src/pages/builders.js [hunk 1]
```diff
@@ -219,7 +219,17 @@ const DeveloperChampionPage = () => {
               <hr />
               <ul>
                 <h4>It is nice to have:</h4>
-                <li>Be an active member of The Reliquary,</li>
+                <li>
+                  Be an active member of{' '}
+                  <a
```
Comment (jerelmiller) [inline src/pages/builders.js:224]: Very minor nit:

Could you use the `Link` component from `@newrelic/gatsby-theme-newrelic` here? We just added the ability to track external page clicks via NR Browser and would love if this were instrumented as well. We get that for free with that component. Could you update to that component?

And just FYI, you can drastically simplify this component as `Link` will pre-fill the `target` and `rel` properties on the anchor tag. Use the following:

```js
import { Link } from '@newrelic/gatsby-theme-newrelic'

<Link to="https://therelicans.com">
  The Relicans
</Link>
```

**Review by jerelmiller — COMMENTED** [review]
Had a very minor suggestion for you! Would you be able to update to use `Link`? Thanks!

File: src/pages/builders.js (hunk 1 above)
Comment (pachicodes) [reply src/pages/builders.js:224]: Sure! Thanks for the tip!

**Comment by pachicodes** [comment]
I have applied the requested change, let me know if it is all right.

**Comment by jerelmiller** [comment]
Hey @pachicodes, thanks for making those changes! It looks like there are some eslint errors that now need to be addressed. Could you take a look at those? I believe that is part of the reason for the failing build. Once you get those fixed, I'd be more than happy to approve!

**Comment by jerelmiller** [comment]
FYI, you can see the ESLint errors by going to the "files" tab in this PR. It will give you a line-by-line breakdown of the failing areas.

**Comment by pachicodes** [comment]
@jerelmiller thanks again for  your guidance and patience! I believe it all good now!

**Review by jerelmiller — APPROVED** [review]
This looks fantastic!! Thanks so much for updating this!!!

**Pull request merged**